src/content/homepage.md

```diff
@@ -40,31 +40,31 @@ technologySection:
   subtitle: What makes Alephium different?
   blockFlowSection:
     title: 'Scalable & Sharded'
-    description: "Alephium is built on a novel and complete sharding algorithm called BlockFlow. It improves on the UTXO model of BTC to make it scalable and uses DAG data structure to reach consensus between different shards. This will allow up to 10,000 Transactions Per Second (currently more than 400 TPS vs Bitcoin's 7 TPS)."
+    description: "Alephium is built on a novel sharding algorithm called BlockFlow. It leverages a stateful UTXO model and DAG data structure to deliver efficient and practical sharding. This enables Alephium to natively support over 10,000 transactions per second while preserving a seamless “'single-chain” user experience."
     links:
       - text: More details
       - text: White paper
         url: https://github.com/alephium/white-paper/blob/master/alephium.pdf
         newTab: true
   smartContractSection:
     title: Programmable & Secure
-    description: 'Alephium introduces the stateful UTXO model offering layer-1 scalability and the same level of programmability as the account model used on Ethereum, whilst being more secure.'
+    description: 'Alephium's stateful UTXO model combines the advantages of the UTXO and account models. It supports mutable states, like those found in Ethereum, for smart contracts while leveraging the security benefits of the UTXO model for assets.
```

Comment:
Be careful with single quotes! The quote you added (Alephium's) will cause issues because the whole string is encapsulated with single quotes. You can encapsulate the string with double quotes to avoid issues.

Reply:
How do I  fix this without removing what's grammatically correct ? Or is it the only way ?